openpype/hosts/houdini/api/lib.py

```diff
@@ -558,19 +558,36 @@ def get_frame_data(node):
         dict: frame data for star, end and steps.
 
     """
-    data = {}
+    if asset_data is None:
+        asset_data = {}
 
-    if node.parm("trange") is None:
+    if log is None:
+        log = self.log
 
-        return data
+    if node.parm("trange") is None:
+        log.debug(
+            "Node has no 'trange' parameter: {}".format(node.path())
+        )
+        return
 
     if node.evalParm("trange") == 0:
-        self.log.debug("trange is 0")
-        return data
+        log.debug(
+            "Node '{}' has 'Render current frame' set. "
+            "Time range data ignored.".format(node.path())
+        )
+        return
```

Comment:
Please make sure to be consistent with your return type - this function is supposed to return a `dict`, so return a `dict`.

Reply:
Tangential, you could declare `data = {}` at the top and return that in each return (which would be an empty `dict`)

Reply:
That was the old code ;) so yes, feel free to revert.